files/ib_perftest.yml

```diff
@@ -0,0 +1,133 @@
+ - job:
+         name: ib_perftest
+         description: "This is the job to run performance tests on the Infiniband Network"
+         project-type: freestyle
+         block-downstream: false
+         concurrent: true
+         properties:
+                 - authorization:
+                         hpc-sig-admin:
+                                 - credentials-create
+                                 - credentials-delete
+                                 - credentials-manage-domains
+                                 - credentials-update
+                                 - credentials-view
+                                 - job-build
+                                 - job-cancel
+                                 - job-configure
+                                 - job-delete
+                                 - job-discover
+                                 - job-move
+                                 - job-read
+                                 - job-status
+                                 - job-workspace
+                                 - ownership-jobs
+                                 - run-delete
+                                 - run-update
+                                 - scm-tag
+                         hpc-sig-devel:
+                                 - job-build
+                                 - job-read
+         parameters:
+                 - node:
+                         name: cluster
+                         allowed-slaves:
+                                 - d05ohpc
+                                 - qdcohpc
+                         allowed-multiselect: False
+                         description: 'Which cluster to use'
+                 - string:
```

Comment:
Nit: reorder the branch arguments to match the others, or it will be slightly confusing when starting jobs by hand. ie client -> automation.